docs/news.rst

```diff
@@ -3,6 +3,204 @@
 Release notes
 =============
 
+.. _release-2.8.0:
+
+Scrapy 2.8.0 (to be released)
+-----------------------------
+
+Deprecation removals
```

Comment:
What do you think about prefacing the release notes with a short summary - this is a maintenance release, with minor features, bug fixes and cleanups?